packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js

```diff
@@ -93,19 +95,28 @@ export default class PciInstancesAddController {
     this.availableRegions = {};
     this.unavailableRegions = {};
 
-    forEach(this.regions, (locationsMap, continent) => {
-      this.availableRegions[continent] = {};
-      this.unavailableRegions[continent] = {};
-      forEach(locationsMap, (datacenters, location) => {
-        [
-          this.availableRegions[continent][location],
-          this.unavailableRegions[continent][location],
-        ] = partition(
-          datacenters,
-          (datacenter) =>
-            !datacenter.isAvailable() ||
-            this.model.flavorGroup.isAvailableInRegion(datacenter.name),
-        );
+    const planCode = `${this.model.flavorGroup.name}.consumption`;
+
+    return this.PciProjectsProjectInstanceService.getProductAvailability(
+      this.projectId,
+      planCode,
+      this.coreConfig.getUser().ovhSubsidiary,
+    ).then((productAvailability) => {
+      const productRegionsAvailables = productAvailability?.plans[0]?.regions;
+
+      forEach(this.regions, (locationsMap, continent) => {
```

Comment:
if duable it's better to use `Array.forEach` instead `Lodash.forEach`.
```suggestion
     this.regions.forEach((locationsMap, continent) => {
```

Reply:
As discuss together, ok for me